### ISSUE TITLE
false positive on `double-slash-comment-whitespace-inside`

### ISSUE THREAD

**Issue by daltones** [kristerkari/stylelint-scss#66]
```scss
$somevariable: 'http://www.example.com';
```
raises:
```
✖  Expected a space after //   scss/double-slash-comment-whitespace-inside
```

`double-slash-comment-whitespace-inside` thinks every `//` is a comment, even if it is inside a string.

(stylelint: 7.0.3 — stylelint-scss: 1.3.3)

**Comment by dryoma** [comment]
Thanks for reporting, please try 1.3.4.
